tasks/bootstrap_local_vm/03_engine_initial_tasks.yml

```diff
@@ -37,6 +37,22 @@
     no_log: true
     with_items:
       - "OVESETUP_CONFIG/adminPassword=str:{{ he_admin_password }}"
+  - name: Add lines to answerfile
+    lineinfile:
+      path: /root/ovirt-engine-answers
+      line: "{{ item }}"
+    no_log: true
+    with_items:
+      - "OVESETUP_DB/password=str:{{ ovirt_engine_setup_db_password | default(omit) }}"
```

Comment:
- `default(omit)` is not needed here, this task will only execute when `ovirt_engine_setup_db_password` is defined
- In order to keep the convention, please change `ovirt_engine_setup_db_password` to start with `he` (something like `he_db_password`)

Reply:
Updated.